docs/deployment-on-arbitrary-cluster.md

```diff
@@ -39,8 +41,19 @@ ovs-vsctl list-br
 ovs-vsctl del-br test-br
 ```
 
+### OpenShift
+
+With OpenShift you can either decide to install Open vSwitch from your
+repositories as described in previous section or using a DaemonSet.
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/kubevirt/ovs-cni/master/examples/openshift-ovs-vsctl.yml
+```
+
 ## Multus
 
+### Kubernetes
+
 Installation of Multus is currently a little tricky. If you have not installed
```

Comment:
what do you mean by tricky?
is there anything else to do than to reorder conf files under ```/etc/cni/net.d``` ?

Reply:
You need to add more configuration, like credentials for multus on Kubernetes API. And the delegate in net.d/00-multus.conf depends on primary network plugin you use.